include/uk/plat/lcpu.h

```diff
@@ -231,8 +241,13 @@ int ukplat_lcpu_run(const __lcpuidx lcpuidx[], unsigned int *num,
 int ukplat_lcpu_wakeup(const __lcpuidx lcpuidx[], unsigned int *num);
 
 #else /* CONFIG_HAVE_SMP */
-#define ukplat_lcpu_idx()	(0)
-#define ukplat_lcpu_count()	(1)
+#define ukplat_lcpu_count()		(1)
+#if !CONFIG_PLAT_KVM
```

Comment:
Please please please, no platform dependencies in generic code 🙏🏼

Reply:
This came as an afterthought. When I first submitted the PR, I was notified that builds were failing on `linuxu`/`xen` with undefined references because they don't have their own definitions of these. So I just quickly hacked in this `#if` to make the CI shut up. :sweat: